pillarbox-player/src/main/java/ch/srgssr/pillarbox/player/PlayerCallbackFlow.kt

```diff
@@ -292,9 +293,12 @@ fun Player.videoSizeAsFlow(): Flow<VideoSize> = callbackFlow {
  *
  * @param defaultAspectRatio Aspect ratio when [Player.getVideoSize] is unknown or audio.
  */
-fun Player.getAspectRatioAsFlow(defaultAspectRatio: Float): Flow<Float> = videoSizeAsFlow().map {
-    it.computeAspectRatio(defaultAspectRatio)
-}
+fun Player.getAspectRatioAsFlow(defaultAspectRatio: Float): Flow<Float> =
+    videoSizeAsFlow()
+        .filterNot { it == VideoSize.UNKNOWN }
+        .map {
+            it.computeAspectRatio(defaultAspectRatio)
+        }
```

Comment:
In the case where the video size is `UNKNOWN` initially, the `Flow` will be empty.
How about adding `.onEmpty { emit(defaultAspectRatio) }` so at least we emit the default value (at specified in the extension documentation)?
```suggestion
        }
        .onEmpty { emit(defaultAspectRatio) }
```

Reply:
Or remove the default parameters and set it when using a StateFlow or a composable State?